### ISSUE TITLE
invokeWithInvocationModeAndOptions method doesn't exist in iOS module

### ISSUE THREAD

**Issue by yongdamsh** [Instabug/instabug-reactnative#179]
This [part](https://github.com/Instabug/instabug-reactnative/blob/master/index.js#L42) guides to use the `invokeWithInvocationModeAndOptions` method in `BugReporting` module. `BugReporting.invokeWithInvocationModeAndOptions` calls same method in the iOS module that doesn't have `invokeWithInvocationModeAndOptions` method.
I think `BugReporting.invokeWithInvocationModeAndOptions` method should process differently depending on OS or add a new method `invokeWithInvocationModeAndOptions` in the iOS module.

Below is my temporary workaround.
```js
// In modules/BugReporting.js file
invokeWithInvocationModeAndOptions: function (invocationMode, invocationOptions) {
  if (Platform.OS === 'ios') {
    Instabug.invokeWithInvocationMode(invocationMode, invocationOptions);
  } else {
    Instabug.invokeWithInvocationModeAndOptions(invocationMode, invocationOptions);
  }
},
```

**Comment by HeshamMegid** [comment]
Thanks for pointing that out! We're working on a fix and will release it very soon.

**Comment by HeshamMegid** [comment]
We just released a fix for this and #180. Please update to 8.0.2.

**Comment by yongdamsh** [comment]
Thanks for the quick fix! 👍
